docs/adr/0044-use-rest-over-gql-when-convenient.md

```diff
@@ -0,0 +1,88 @@
+B. Use auth-client for all new network requests, or prefer auth-client and only use GQL where already set up
+C. Use auth-client for all new network requests, and plan to strip out GQL and use auth-client everywhere
+
+## Decision Outcome
+
+Chosen option: B, "Use auth-client for all new network requests, or prefer auth-client and only use GQL where already set up" because by standardizing on auth-client for now or at least using it when it is convenient, we minimize overhead in setting up and maintaining GQL mutations/resolvers while leaving open the possibility of moving to a framework like NextJS later, where we can reevaluate our data-fetching and state strategies more holistically.
+
+This options allow engineers the most flexibility to get tasks out the quickest way, especially if we can create a DRY wrapper for auth-client like what's described in [FXA-10861](https://mozilla-hub.atlassian.net/browse/FXA-10861).
+
+Additionally, while we started a [state management/architecture doc](https://docs.google.com/document/d/17dCwjxhxBKXDclkydmPHD91Pj3suIFihR6fZw6m126Q/edit?tab=t.0#heading=h.3neqekw0gen1) in 2023, we should revisit and better finish this documentation to clarify when to use which tool for state management.
+
+### Positive Consequences
+
+- Clarity of direction while maximizing flexibility for developers for now
+- Reduced complexity and cognitive load
+- Increased developer velocity due to not needing to set up GQL resolvers
+- By using auth-client we'll be directly querying auth-server instead of essentially using our GraphQL server as a proxy to talk to auth-server for us (which seems preferable especially for simple mutations)
+
+### Negative Consequences
+
+- We'll still have to manually update Apollo Cache
+- Potential loss of GQL benefits, including previous GQL refactoring investments
+- We'll continue to have a mixed usage of GQL and auth-client in our client-side with no immediate plans to mitigate the inconsistency
+- If we change our minds later, we'll have more factoring work to do to deprecate auth-client. However, it's worth noting FxA already has a very mixed usage of auth-client and GQL
+
+## Pros and Cons of the Options
+
+### Option A: Use GQL for all new network requests, continue creating GQL resolvers that use auth-client, and plan to deprecate auth-client through client-side and server-side refactoring
+
+- Good, because we'll finally be able to realize the full benefits of using GQL
+- Good, because it eliminates the complexity of maintaining two data-fetching paradigms, simplifying the codebase to a single approach
+- Good, because we want to do some back-end refactoring to use `libs/` regardless
+- Bad, because moving away from GQL later if desired may be challenging
+- Bad, because the level of effort to complete the back-end refactor is high and also requires front-end refactoring away from auth-client, both of which may not be useful to us if we move to NextJS in the future
```

Comment:
I'd also note that using GQL in this way (vs skipping auth-client and auth-server and directly connecting to libs)  adds unnecessary development complexity (significantly increasing dev time), introduces additional points of failure, increases latency and complicates error-handling.